ocm/__main__.py

```diff
@@ -0,0 +1,229 @@
+import argparse
+import dataclasses
+import datetime
+import json
+import os
+import sys
+
+import ocm
+
+
+try:
+    import yaml
+    _have_yaml = True
+except ImportError:
+    _have_yaml = False
+
+try:
+    import oci
+    import oci.auth
+    import oci.client
+    _have_oci = True
+except ImportError:
+    _have_oci = False
+
+
+def _parse_yaml_or_json(path) -> dict:
+    with open(path) as f:
+        if _have_yaml:
```

Comment:
Shouldn't we rather decide whether to you `yaml` or `json` based on file contents (of course only if `yaml` module is available)?

Reply:
well, it is always safe to use `yaml`, as each json-document can always also be parsed using a yamlparser. I wanted to at least _allow_ usage of ocm-package w/o installed pyyaml package (not sure whether this will really ever be a reasonable use-case).
we might do a pre-check and prefer json if document is json-compliant; however, the only disadvantage I see from always using yaml if it is present is a slightly worse performance (which is not actually a concern for us at this point, I think)